nitro-protocol/test/contracts/TrivialApp/requireStateSupported.test.ts

```diff
@@ -98,10 +95,14 @@ describe('requireStateSupported', () => {
     const from: RecoveredVariablePart = mockSigs(getVariablePart(fromState));
     const to: RecoveredVariablePart = mockSigs(getVariablePart(toState));
 
-    const tx = trivialApp.requireStateSupported(getFixedPart(fromState), [from], to);
+    const requireStateSupported = await trivialApp.requireStateSupported(
```

Comment:
Perhaps a better name for this variable? `result`?

Reply:
Agree with you. Also have noticed, the comment on why we are checking the length of `result` is not 100% clear, so I will tidy them also.